tensorpack/utils/viz.py

```diff
@@ -242,6 +243,34 @@ def dump_dataflow_images(df, index=0, batched=True,
                 vizlist = vizlist[vizsize:]
 
 
+def intensity_to_rgb(intensity, cmap='jet', normalize=False):
```

Comment:
There was a reason for [not using JET](https://jakevdp.github.io/blog/2014/10/16/how-bad-is-your-colormap/) in my code.